client/container_wait_test.go

```diff
@@ -127,3 +128,36 @@ func ExampleClient_ContainerWait_withTimeout() {
 		log.Fatal(err)
 	}
 }
+
+type TestClient_ContainerWait_cancelReader struct{}
+
+func (TestClient_ContainerWait_cancelReader) Read(p []byte) (n int, err error) {
```

Comment:
Minor nit; perhaps a shorter name would work for this, which can be unexpected. e.g. `cancelReader`